components/app/Footer.tsx

```diff
@@ -0,0 +1,102 @@
+import Link from 'next/link'
+
+function Content() {
+  return (
+    <section>
```

Comment:
I personally never know when to use `<section>`, what's the reasoning here? :)

Reply:
Yes, I copied it over from the current site 😬